diff --git a/test-fixtures/Makefile.am b/test-fixtures/Makefile.am
index 62fd9d1..e9e80f5 100644
--- a/test-fixtures/Makefile.am
+++ b/test-fixtures/Makefile.am
@@ -11,6 +11,75 @@ if !USE_GLIB
 libtest_fixtures_la_CPPFLAGS += -I$(top_builddir)/deps/glib
 endif
 
+if USE_GLIB
+
+# public api headers
+cogl_public_h = \
+	$(srcdir)/cogl-attribute-buffer.h 	\
+	$(srcdir)/cogl-attribute.h 		\
+	$(srcdir)/cogl-bitmap.h 		\
+	$(srcdir)/cogl-buffer.h 		\
+	$(srcdir)/cogl-clip-state.h		\
+	$(srcdir)/cogl-clutter.h       		\
+	$(srcdir)/cogl-color.h 			\
+	$(srcdir)/cogl-context.h 		\
+	$(srcdir)/cogl-depth-state.h 		\
+	$(srcdir)/cogl-display.h 		\
+	$(srcdir)/cogl-error.h			\
+	$(srcdir)/cogl-euler.h 			\
+	$(srcdir)/cogl-framebuffer.h		\
+	$(srcdir)/cogl-gles2-types.h		\
+	$(srcdir)/cogl-gles2.h			\
+	$(srcdir)/cogl-index-buffer.h 		\
+	$(srcdir)/cogl-indices.h 		\
+	$(srcdir)/cogl-matrix.h 		\
+	$(srcdir)/cogl-meta-texture.h		\
+	$(srcdir)/cogl-object.h 		\
+	$(srcdir)/cogl-offscreen.h 		\
+	$(srcdir)/cogl-onscreen-template.h 	\
+	$(srcdir)/cogl-frame-info.h		\
+	$(srcdir)/cogl-onscreen.h		\
+	$(srcdir)/cogl-pipeline-layer-state.h 	\
+	$(srcdir)/cogl-pipeline-state.h 	\
+	$(srcdir)/cogl-pipeline.h 		\
+	$(srcdir)/cogl-pixel-buffer.h		\
+	$(srcdir)/cogl-poll.h			\
+	$(srcdir)/cogl-primitive-texture.h	\
+	$(srcdir)/cogl-primitive.h 		\
+	$(srcdir)/cogl-output.h			\
+	$(srcdir)/cogl-quaternion.h 		\
+	$(srcdir)/cogl-matrix-stack.h		\
+	$(srcdir)/cogl-renderer.h 		\
+	$(srcdir)/cogl-snippet.h		\
+	$(srcdir)/cogl-sub-texture.h            \
+	$(srcdir)/cogl-texture-2d-gl.h 		\
+	$(srcdir)/cogl-texture-2d-sliced.h      \
+	$(srcdir)/cogl-texture-2d.h             \
+	$(srcdir)/cogl-texture-3d.h             \
+	$(srcdir)/cogl-texture-rectangle.h      \
+	$(srcdir)/cogl-texture.h 		\
+	$(srcdir)/cogl-types.h 			\
+	$(srcdir)/cogl-vector.h 		\
+	$(srcdir)/cogl-fence.h       		\
+	$(srcdir)/cogl-version.h		\
+	$(srcdir)/cogl.h			\
+	$(NULL)
+
+glib_enum_headers = $(cogl_public_h)
+
+CLEANFILES =
+DISTCLEANFILES =
+
+BUILT_SOURCES =
+EXTRA_DIST =
+
+# glib-mkenums rules
+glib_enum_h = cogl-enum-types.h
+glib_enum_c = cogl-enum-types.c
+glib_enum_headers = $(cogl_public_h)
+include $(top_srcdir)/build/autotools/Makefile.am.enums
+endif
+
 libtest_fixtures_la_CPPFLAGS += \
 	-DCOGL_DISABLE_DEPRECATED \
 	-DTESTS_DATADIR=\""$(top_srcdir)/tests/data"\" \
